src/charm.py

```diff
@@ -398,8 +398,10 @@ def _request_certificates(self):
 
         bind_addrs = kubernetes_snaps.get_bind_addresses()
         common_name = kubernetes_snaps.get_public_address()
+        ingress_ip = kubernetes_snaps.get_ingress_ip()
```

Comment:
ok, hold... i don't see this function in `kubernetes_snaps`.
@sombrafam did you hallucinate this function existing?

Reply:
I also added it in this PR: https://github.com/charmed-kubernetes/charm-lib-kubernetes-snaps/pull/36